### ISSUE TITLE
Sourcing for carrot info

### ISSUE THREAD

**Issue by dmil** [dmil/my-simple-website#14]
Let's link to a reputable source when it comes to carrots. Don't modify the paragraph text though, that is a separate ticket (#15). This ticket is only about improving the copy of the paragraph. What can we do to improve the copy and make it both informative and witty?

**Comment by KaleighRogers** [comment]
Yeah, there is a link there but I think I can find something a little more reputable.

**Comment by dmil** [comment]
I think business insider has an article about this. we could use that link

**Comment by nrakich** [comment]
I think Carrot Insider is a more reputable source, tbh

**Comment by ameliatd** [comment]
I think the Carrot Museum is really where we should go: http://www.carrotmuseum.co.uk/maroon.html
